www/webapp/src/views/CrudList.vue

```diff
@@ -197,6 +197,7 @@
                 :key="id"
                 :readonly="column.readonly"
                 :disabled="$store.getters.working || itemIsReadOnly(itemFieldProps.item)"
+                :placeholder="column.placeholder"
```

Comment:
This will also get accessed for column types that don't have that attribute. (You added it to `GenericText` only.) 

I invented `fieldProps` to deal with this case ;-) I'll comment in `TokenList.vue` how that works, but you can remove the line here.

Reply:
fixed